input.cpp

```diff
@@ -986,7 +989,7 @@ typedef struct
 	int      accx, accy;
 }  devInput;
 
-static devInput input[NUMDEV] = { 0 };

```

Comment:
Since C++11 you zero-initialize a struct/class with empty braces, Clang complains here as it only partially initializes the struct.

 - https://en.cppreference.com/w/cpp/language/zero_initialization

Reply:
In C standard, initialization with { 0 } means initialize all members with 0. So it's pretty much standard way.

Reply:
I'm not sure the change to {} will initialize it with 0 if compiled in GCC. This must be verified before change.

Reply:
@sorgelig I checked it with `ARM gcc 5.4` on godbolt and it produces the same instructions. https://godbolt.org/z/Fh1vKr

Reply:
good.
I didn't know about this service.